.travis.yml

```diff
@@ -6,12 +6,12 @@ install:
 
 script:
   - tox -e py27
-  - cd core && tox -e py27
+  - (cd core && tox -e py27)
   - tox -e py34
-  - cd core && tox -e py34
+  - (cd core && tox -e py34)
   - tox -e lint
   - tox -e cover
-  - cd core && tox -e cover
+  - (cd core && tox -e cover)
```

Comment:
Can you be more specific? This runs `tox -e cover` in the core subpackage. There are changes to `tox.ini` and `core/tox.ini`. The changes to the umbrella `tox.ini` are so that core coverage can get pulled in.

It's worth running `cover` on each standalone package. As evidenced by https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2367/commits/3ff2b5807e7c6325a69886136ded470e905c05c7, some coverage may come transitively via other usage, so we should figure out how the standalone module is missing coverage on its own.